api/nodes/serializers.py

```diff
@@ -72,8 +72,8 @@ class NodeSerializer(JSONAPISerializer):
     id = IDField(source='_id', read_only=True)
     type = TypeField()
 
-    category_choices = Node.CATEGORY_MAP.keys()
-    category_choices_string = ', '.join(["'{}'".format(choice) for choice in category_choices])
+    category_choices = Node.CATEGORY_MAP.items()
```

Comment:
OPTIONS request to /v2/nodes/ after this change:

![screen shot 2016-03-16 at 3 26 50 pm](https://cloud.githubusercontent.com/assets/9755598/13825937/6515d3da-eb8b-11e5-9859-889a552cb193.png)